internal/pkg/print/print.go

```diff
@@ -171,7 +171,12 @@ func (p *Printer) PagerDisplay(content string) error {
 	if outputFormat == NoneOutputFormat {
 		return nil
 	}
-	pagerCmd := exec.Command("less", "-F", "-S", "-w")
+
+	// less arguments
+	// -F: exits if the entire file fits on the first screen
+	// -S: disables line wrapping
+	// -R: interprets ANSI color and style sequences
+	pagerCmd := exec.Command("less", "-F", "-S", "-R")
```

Comment:
the `-w` argument was removed because it is used to wrap lines longer than the width of the terminal window, but we are disabling line wrapping with `-S` and that's the intended behavior

Reply:
`-w` was used to highlight the first line after moving one full page down (with the `f` key)
![image](https://github.com/stackitcloud/stackit-cli/assets/28832884/c7a72085-c921-4eca-bb17-24e33cca3c9e)

Reply:
I guess I read some wrong documentation then, re-added it